repo/packages/O/openvpn/1/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
please create a PR against dcos/examples repo to add an example documentation for this package. It is also a requirement now to submit a package to Universe. You can you the template from other examples there.

Reply:
https://github.com/dcos/examples/pull/214